src/blocks/mrc_class_method_def.ts

```diff
@@ -337,12 +337,13 @@ export const setup = function() {
   Blockly.Blocks[PARAM_CONTAINER_BLOCK_NAME] = METHOD_PARAM_CONTAINER;
 };
 
-import { Order, PythonGenerator } from 'blockly/python';
+import { Order } from 'blockly/python';
```

Comment:
Do these imports have to be here? Can they be at the top of the file? https://google.github.io/styleguide/tsguide.html#source-file-structure

Reply:
Sure.